UnityProject/Assets/Scripts/Core/Editor/EditorDrawPreview.cs

```diff
@@ -47,6 +47,11 @@ private static void DrawProjectItem(string guid, Rect selectionRect)
 			if (Dictionaryguid.ContainsKey(guid))
 			{
 				mainTex = GetCorrectTexture(Dictionaryguid[guid]);
+
+				if (mainTex == null)
+				{
+					Debug.LogError($"Sprite SO {sTing} has null value");
```

Comment:
Use this instead
```suggestion
					Logger.LogError($"Sprite SO {sTing} has null value");
```

Reply:
Done, going to do some other changes, and I'll commit